css/types/image.json

```diff
@@ -977,6 +979,59 @@
                 }
               }
             },
+            "multiposition": {
+              "__compat": {
+                "description": "Multi-position color stops",
+                "support": {
+                  "chrome": {
+                    "version_added": "71"
```

Comment:
I need to verify that this actually shipped in 71. I'm having trouble with my copy of Chrome beta just now and I can't find this in the code base.

Reply:
I have 70 and 72 now. 72 works. In 70 doesn't.